packages/chart/src/SummaryC.ts

```diff
@@ -162,8 +174,16 @@ export interface SummaryC {
     paddingSizeExp(_: number): this;
     iconSizeMult(): number;
     iconSizeMult(_: number): this;
+
+    centeredLabel(): boolean;
+    centeredLabel(_: boolean): this;
+    centeredValue(): boolean;
+    centeredValue(_: boolean): this;
+
 }
 
+SummaryC.prototype.publish("centeredValue", false, "boolean", "Horizontally centers the value if true");
+SummaryC.prototype.publish("centeredLabel", false, "boolean", "Horizontally centers the label if true");
```

Comment:
How about:
```
...publish("valueAnchor", "start", "set", "Anchor Position", ["start", "middle", "end"], { tags: ["Intermediate"] });
...publish("labelAnchor", "start", "set", "Anchor Position", ["start", "middle", "end"], { tags: ["Intermediate"] });
```
To be consistent with regular "Text" widget?

Reply:
I should have thought of this